content/docs/setup/kubernetes/upgrading-istio.md

```diff
@@ -89,3 +86,20 @@ of sidecar proxy. There are two cases: Manual injection and Automatic injection.
 
     There are some tricks to reload all pods. E.g. There is a [bash script](https://gist.github.com/jmound/ff6fa539385d1a057c82fa9fa739492e)
     which triggers the rolling update by patching the grace termination period.
+
+### Migrating to the new networking APIs
+
```

Comment:
should we move this section earlier under ### sidecar upgrade? the proxyimage annotation is useful for auto injection and manual injection (I think).

Reply:
I think there are different things - first you upgrade your existing app and install to 0.8, including injecting to 0.8 v1 sidecar.

We do need more content on how to use the conversion tool - I just mentioned how to upgrade an individual pod to proxyv2, this should link or include the info on how to convert the config.

Reply:
With auth you get the following error about nodePort collision, this doc already improves the state of the art so I will work on fixing the other part.

```
Error from server (Invalid): error when applying patch:
{"metadata":{"annotations":{"kubectl.kubernetes.io/last-applied-configuration":"{\"apiVersion\":\"v1\",\"kind\":\"Service\",\"metadata\":{\"annotations\":{},\"labels\":{\"chart\":\"ingress-0.8.0\",\"heritage\":\"Tiller\",\"istio\":\"ingress\",\"release\":\"RELEASE-NAME\"},\"name\":\"istio-ingress\",\"namespace\":\"istio-system\"},\"spec\":{\"ports\":[{\"name\":\"http\",\"nodePort\":32000,\"port\":80},{\"name\":\"https\",\"port\":443}],\"selector\":{\"istio\":\"ingress\"},\"type\":\"LoadBalancer\"}}\n"},"labels":{"chart":"ingress-0.8.0","heritage":"Tiller","release":"RELEASE-NAME"}},"spec":{"$setElementOrder/ports":[{"port":80},{"port":443}],"ports":[{"nodePort":32000,"port":80}]}}
to:
&{0xc420abd140 0xc420158620 istio-system istio-ingress install/kubernetes/istio-auth.yaml 0xc42149ec60 0xc42149ee50 9964381 false}
for: "install/kubernetes/istio-auth.yaml": Service "istio-ingress" is invalid: spec.ports[0].nodePort: Invalid value: 32000: provided port is already allocated
Error from server (Invalid): error when creating "install/kubernetes/istio-auth.yaml": Service "istio-ingressgateway" is invalid: spec.ports[0].nodePort: Invalid value: 31380: provided port is already allocated
```

Reply:
Node port should be off by default - only used for testing (or special cases)